rosdep/base.yaml

```diff
@@ -226,6 +226,14 @@ beep:
   fedora: [beep]
   gentoo: [app-misc/beep]
   ubuntu: [beep]
+benchmark:
+  debian:
+    stretch: null
+    '*': [libbenchmark-dev]
+  fedora: [google-benchmark]
+  ubuntu:
+    xenial: null
+    '*':[libbenchmark-dev]
```

Comment:
CI is failing.  There definitely needs to be a space between `'*'` and the value.  Looking at all of the current rules, it looks like it *may* also be the case that the `*` has to come first in the list.  Can you make those two changes and see if CI is more happy?

Reply:
Oh, that missed blank space was a typo.

I've updated the PR with the changes.